app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  require "will_paginate/array"
+  PAGE_SIZE = 4
+
+  def index
+    page = (params[:page] || '1').to_i
+
+    client = Tumblr::Client.new({
```

Comment:
We could probably teach them, but they wouldn't be able to preview changes and there would be no toolbar for styling.